src/oce_ale.F90

```diff
@@ -2054,6 +2053,7 @@ subroutine compute_hbar_ale(dynamics, partit, mesh)
     if (.not. trim(which_ALE)=='linfs') then
 !$OMP PARALLEL DO
         do n=1,myDim_nod2D
+            if (ulevels_nod2D(n)>1) cycle
```

Comment:
this change here affects the zstar case the upper change in line 1931 will affect the linfs case ?

Reply:
Yes.